src/controllers/organizationController.ts

```diff
@@ -65,8 +72,52 @@ export default class OrganizationController {
       }
       return withValidatedOrValidationError(
         OrganizationRegistrationParams.decode(req.body),
-        (organizationRegistrationParams: OrganizationRegistrationParams) =>
-          registerOrganization(organizationRegistrationParams, user)
+        async (
+          organizationRegistrationParams: OrganizationRegistrationParams
+        ) => {
+          const errorResponseOrSuccessResponse = await registerOrganization(
+            organizationRegistrationParams,
+            user
+          );
+          return errorResponseOrSuccessResponse.map(async response => {
+            const organization = response.payload;
+            const outputFolder = `./documents/${organization.ipa_code}`;
```

Comment:
add a todo comment linked to pivotal task: files must be stored on cloud storage (azure blob)